src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let amount = days * 40;
+
+  if (days >= 3 && days < 7) {
+    amount -= 20;
+  } else if (days >= 7) {
+    amount -= 50;
```

Comment:
Great job! I recommend you move these numbers to variables with appropriate names. For example, the number 50 could be stored in a variable with the name sevenDaysDiscount.